paper-slider.html

```diff
@@ -292,7 +292,7 @@
      * @method increment
      */
     increment: function() {
-      this.value = this._clampValue(this.value + this.step);
+      this.value = this._clampValue(this.value + this.step).toString();
```

Comment:
That seems like a potential bug. I would want to ask @blasten about that unless it's intentional.